analytics-integrations/moengage/src/main/res/layout/moe_image_message_button.xml

```diff
@@ -0,0 +1,63 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
These are required for the in app messages, which are a part of our library, i intentionally did not ship it in an aar because provide our clients an ability to customize them as well.